ci/nightly/pipeline.yml

```diff
@@ -532,6 +533,17 @@ steps:
       - ./ci/plugins/cloudtest:
           args: [-m=long,  --aws-region=us-east-1, test/cloudtest/test_upgrade.py]
 
+  - id: node-failure-recovery
```

Comment:
This needs to have k8s in the name of the step and the label, so that people are never confused about what "nodes" this is all about.

Reply:
✅